src/components/YahooSellModal.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { makeStyles } from '@material-ui/core/styles'
+import DashboardPopover from 'src/components/DashboardPopover'
+import yahooSell from 'src/assets/images/yahooSell.png'
+import Typography from '@material-ui/core/Typography'
+import Button from '@material-ui/core/Button'
+import SetYahooSearchOptInMutation from 'src/utils/mutations/SetYahooSearchOptInMutation'
+import SetUserSearchEngineMutation from 'src/utils/mutations/SetUserSearchEngineMutation'
+import CreateSearchEnginePromptLogMutation from 'src/utils/mutations/CreateSearchEnginePromptLogMutation'
+
+const useStyles = makeStyles((theme) => ({
+  popoverText: {
+    marginLeft: theme.spacing(3),
+    marginRight: theme.spacing(3),
+    marginTop: theme.spacing(3),
+  },
+  popover: {
+    maxWidth: theme.spacing(72),
+  },
+  popoverPaper: {
+    borderRadius: theme.spacing(1.5),
+  },
+  acceptButton: {
+    background: '#29BEBA',
+    borderRadius: '15px',
+    height: '30px',
+    fontWeight: '800',
+    fontFamily: 'Poppins',
+    marginLeft: theme.spacing(1),
+    color: 'white',
+  },
+  declineButton: {
+    marginLeft: 'auto',
+    borderRadius: '15px',
+    height: '30px',
+    fontWeight: '500',
+    fontFamily: 'Poppins',
+  },
+  buttons: {
+    width: '100%',
+    marginTop: theme.spacing(3),
+    marginBottom: theme.spacing(3),
+    display: 'flex',
+    justifyContent: 'flex-end',
+  },
+  title: {
+    fontWeight: '700',
+  },
+}))
+
+const YahooSellModal = ({
+  hardSell,
+  userId,
+  anchorEl,
+  onAccept,
+  onDecline,
+}) => {
+  const classes = useStyles()
+  const [open, setOpen] = useState(true)
+  const declineSearchOptIn = () => {
+    CreateSearchEnginePromptLogMutation(userId, 'Yahoo', false)
+    onDecline()
+    setOpen(false)
+  }
+  const acceptSearchOptIn = () => {
+    SetYahooSearchOptInMutation(userId, true)
+    SetUserSearchEngineMutation(userId, 'Yahoo')
+    CreateSearchEnginePromptLogMutation(userId, 'Yahoo', true)
+    onAccept()
+    setOpen(false)
+  }
+  return (
+    <DashboardPopover
```

Comment:
You might want to make this a proper [modal](https://mui.com/components/modal/#main-content) so that it includes the backdrop, etc.?
https://www.figma.com/file/3eY3CAnOdBiQ7eFOP8rnp9/Tab-Legacy%3A-Yahoo-Partnership?node-id=146%3A12570

Ignore if you have a different UI approach in mind.